### PR TITLE
Anushka: Add assignments

### PR DESCRIPTION
**Pull request by Anushka014**

### CODE REVIEW

File: QA Interns 2020/QA Materials/anushka.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Reading materials I used extra than the Ron-patton-software-testing pdf :
+1) Bug bash testing (https://www.quora.com/What-is-a-bug-bash-testing).
+2) Edge case (https://teamtreehouse.com/library/happy-path-vs-testing-edge-cases).
+3)Mean time between failure(https://limblecmms.com/blog/mttr-mtbf-mttf-guide-to-failure-metrics/).
+4)Principle of software testing(https://www.geeksforgeeks.org/software-engineering-seven-principles-of-software-testing/).
+5)Agile vs Waterfall model(https://www.guru99.com/waterfall-vs-agile.html).
+6)Agile vs V model (https://www.geeksforgeeks.org/difference-between-agile-model-and-v-model/).
+7)Boundary value and Equivalence partioning.(https://www.guru99.com/equivalence-partitioning-boundary-value-analysis.html).
+8)Decision table testing.(https://www.guru99.com/decision-table-testing.html).
```
Comment (prertik) [inline QA Interns 2020/QA Materials/anushka.txt]: Unnecessary change.

**Comment by prertik** [comment]
https://github.com/Nischalkhadka/Quality-Assurance/pull/6#issuecomment-573582573

And use present tense while committing, like: `Add materials`

**Comment by Anushka014** [comment]
Amend commit message please review

File: QA Interns 2020/Assignments/Anushka Shrestha/Test.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+		{
+			"key": "bearer_token",
+			"value": "",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2020-01-17T04:46:11.468Z",
+	"_postman_exported_using": "Postman/7.14.0"
+}
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/Anushka Shrestha/Test.postman_environment.json]: Add a new line here

File: QA Interns 2020/Assignments/Anushka Shrestha/Api Test.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+				"id": "13cb3312-b452-4820-9fc0-5acfaf6ade54",
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		}
+	],
+	"protocolProfileBehavior": {}
+}
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/Anushka Shrestha/Api Test.postman_collection.json]: Add new line

File: QA Interns 2020/Assignments/Anushka Shrestha/Answer.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+"errorCode":"VALIDATION_ERROR_INVOICE_SUPPLIER_TAX_FEWER_THAN_TOTAL_OFLINE_ITEM_SUPPLIER_TAXES",
+"groupId":null,
+"messageKey":null,
+"message":"Invoice Supplier Tax is less than the total of all Invoice line items Supplier Taxes",
+"messageArguments":null,
+"context":{ 
+
+}
+}
+]
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/Anushka Shrestha/Answer.json]: Add new line

File: QA Interns 2020/Assignments/Anushka Shrestha/Test.postman_environment.json (hunk 2 above)
Comment (Anushka014) [reply QA Interns 2020/Assignments/Anushka Shrestha/Test.postman_environment.json]: Dai bujhena k vanu vaako?

**Comment by Anushka014** [comment]
Done. Please review
